### ISSUE TITLE
[Feature request] Nice to have option to support arranging segments placement within the timeline

### ISSUE THREAD

**Issue by markusd1984** [mifi/lossless-cut#596]
Nice to have option to move parts that appear later in the video earlier or vice versa, without having to export segments and create a merge. (advanced option, similar to what we are used to in video editors)

https://github.com/mifi/lossless-cut/issues/284#issuecomment-759415562

**Comment by mifi** [comment]
I think what would be the ultimate solution would be to start with one big sgement, then be able to chop up, duplicate and move segments around like in a normal editor

**Comment by markusd1984** [comment]
> I think what would be the ultimate solution would be to start with one big sgement, then be able to chop up, duplicate and move segments around like in a normal editor

Exactly. (If your thinking of creating a new app you can call it LosslessEditor :D )

**Comment by mifi** [comment]
Closing this as I have gathered info from similar issues like this one and created one feature requests for this with more info: #976
